tailcfg/tailcfg.go

```diff
@@ -1852,11 +1852,8 @@ const (
 	// of connections to the default network interface on Darwin nodes.
 	CapabilityDebugDisableBindConnToInterface = "https://tailscale.com/cap/debug-disable-bind-conn-to-interface"
 
-	// CapabilityTailnetLockAlpha indicates the node is in the tailnet lock alpha,
-	// and initialization of tailnet lock may proceed.
-	//
-	// TODO(tom): Remove this for 1.35 and later.
-	CapabilityTailnetLockAlpha = "https://tailscale.com/cap/tailnet-lock-alpha"
+	// CapabilityTailnetLock indicates the node may initialize tailnet lock.
+	CapabilityTailnetLock = "https://tailscale.com/cap/tailnet-lock-alpha"
```

Comment:
I don't think we can change this unfortunately - because its used by the client to see if it can enable tailnet lock, and old versions will always exist expecting the alpha suffix.

Reply:
I guess the alternative is we send both down? or both down till 1.46?

Reply:
Is this to/from control? It could figure out the correct name from map version.

Reply:
Gotcha. Its a little too late in the day for a lockstep change but I'll do that on Monday.